.github/workflows/forbid-clusterpolicies.yaml

```
@@ -0,0 +1,29 @@
+name: Forbid ClusterPolicies outside Policies component
+
+on:
+  pull_request:
+    branches: [ main ]
+    paths:
+      - 'components/**'
+      - '!components/policies/**'
+
+jobs:
+  forbid-clusterpolicies:
+    name: Forbid ClusterPolicies outside Policies component
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Kustomize
+        uses: multani/action-setup-kustomize@v1
+        with:
+          version: 5.6.0
+
+      - name: Run forbid-clusterpolicies script
+        env:
+          OUTPUT: GITHUB
+        run: |
+          find argo-cd-apps components -name 'kustomization.yaml' \
+            ! -path 'components/policies/*' \
+            | \
+            xargs -I {} -n1 -P8  bash -c './hack/forbid-clusterpolicies.sh $(dirname "{}")'
```

I don't think we get 8 cores in github actions.

```suggestion
            xargs -I {} -n1 "-P$(nproc)"  bash -c './hack/forbid-clusterpolicies.sh $(dirname "{}")'
```

---

we do not get 8 cores but we can still do multi threading, 8 seems good to me